plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisConnectorConfig.java

```diff
@@ -220,13 +221,27 @@ public RedisConnectorConfig setRedisPassword(String redisPassword)
         return this;
     }
 
+    public boolean isUseSsl()
+    {
+        return useSsl;
+    }
+
+    @Config("redis.use-ssl")
+    @ConfigSecuritySensitive
```

Comment:
Remove unnecessary `@ConfigSecuritySensitive` annotation. It's just a boolean flag.